source/dub/data/settings.d

```diff
@@ -171,3 +172,90 @@ unittest {
      auto m3 = Settings.init.merge(c1);
      assert(m3 == c1);
 }
+
```

Comment:
Shall I instead parameterize the unittest on settings key being either `customCachePaths` or `overrideCachePaths`?